README.md

```diff
@@ -109,6 +109,28 @@ end
 to prevent accidental data loss. After successful or failed LHM migrations, these leftover
 tables must be cleaned up.
 
+### Usage with ProxySQL
+LHM has features to recover from connexion loss that are "ProxySQL-aware". There are multiple ways that connection loss could
+induce data loss. Therefore, if LHM is used with ProxySQL, it will check that the MySQL host is consistent across the schema migrations. 
+Here are the required steps to use LHM with ProxySQL:
+1) Ensure that the ActiveRecord::QueryLogs context is set (requires ActiveRecord >= 7.0.0.aplha2). Ex:
+```ruby
+ActiveRecord::QueryLogs.set_context(maintenance: "lhm") do
+   Lhm.setup(conn)
+   // Run LHM
+end
+``` 
+_Note: Other Query annotations could work (i.e. Magnolia), but this feature is only tested with ActiveRecord::QueryLogs_
+2) Add a `mysql_query_rule`:
+```
+{
+  rule_id = 4
+  active = 1
+  match_pattern = "maintenance:lhm"
+  destination_hostgroup = <MySQL writer's hostgroup>
+}
+```
+
```

Comment:
Could we make this description a bit more clearer. For eg, it is mentioned here that one should add a query rule in proxysql to make Lhm work, but the reason is not described.

Reply:
Good catch! Will do!